MdeModulePkg/MdeModulePkg.dec

```diff
@@ -198,7 +198,7 @@
   ## @libraryclass  This library is used to report memory type information change
   ##                across a reboot.
   MemoryTypeInformationChangeLib|Include/Library/MemoryTypeInformationChangeLib.h
-  
+

```

Comment:
may as well not change this here, same with other whitespace changes, just for easier rebasing